styles/style.css

```diff
@@ -0,0 +1,124 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #e5e5e5;
+}
+
+/* Global Styles */
+ul,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+ul li {
+  list-style: none;
+}
+
+/* Navigation Styles */
+.navbar {
+  width: 100%;
+  min-height: 48px;
+  max-height: 48px;
+  background: #fff;
+  padding: 0 24px;
+
+  /* Layout */
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
```

Comment:
[OPTIONAL] Kindly fix the navbar. Right now, it looks like below. "I am Simon" is overlayed by the fixed navbar. You can give some margin to the main section.
![Screen Recording 2022-12-02 at 12 12 29 AM](https://user-images.githubusercontent.com/25887253/205161674-5409b9d1-fa16-42ea-a48f-26d6fc181648.gif)